src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -112,11 +121,14 @@ private async Task<(Document document, bool addedFields)> GenerateThisOrBaseDele
                 return default;
             }
 
-            private async Task<(Document document, bool addedFields)> GenerateThisOrBaseDelegatingConstructorAsync(int argumentCount)
+            private async Task<(Document document, bool addedFields)> GenerateThisOrBaseDelegatingConstructorAsync(
+                int argumentCount, NamingRule fieldNamingRule)
             {
                 (Document document, bool addedField) edit;
-                if ((edit = await GenerateDelegatingConstructorAsync(argumentCount, _state.TypeToGenerateIn).ConfigureAwait(false)).document != null ||
-                    (edit = await GenerateDelegatingConstructorAsync(argumentCount, _state.TypeToGenerateIn.BaseType).ConfigureAwait(false)).document != null)
+                if ((edit = await GenerateDelegatingConstructorAsync(argumentCount, _state.TypeToGenerateIn,
+                        fieldNamingRule).ConfigureAwait(false)).document != null ||
+                    (edit = await GenerateDelegatingConstructorAsync(argumentCount, _state.TypeToGenerateIn.BaseType,
+                        fieldNamingRule).ConfigureAwait(false)).document != null)
```

Comment:
💡 This is really hard to read. One of the following approaches would be preferable:

1. Keep this expression on two lines, and just add the new argument
2. Break up this `if` statement into two consecutive `if` statements

    ```
    if (firstcase)
    {
      return edit;
    }

    if (secondcase)
    {
      return edit;
    }
    ```

Reply:
Agreed.  I went with the longer lines, even though they're pretty long for github.